yellowbrick/bestfit.py

```diff
@@ -95,11 +96,11 @@ def draw_best_fit(X, y, ax, estimator='linear', **kwargs):
 
     # Estimators are the types of best fit lines that can be drawn.
     estimators = {
-        LINEAR: fit_linear,               # Uses OLS to fit the regression
-        QUADRATIC: fit_quadratic,         # Uses OLS with Polynomial order 2
-        EXPONENTIAL: fit_exponential,     # Not implemented yet
-        LOG: fit_log,                     # Not implemented yet
-        SELECT_BEST: fit_select_best,     # Selects the best fit via MSE
```

Comment:
Bleh, kind of annoying that Black did this - it usually lines up comments, but I guess not in a dictionary.